test/fixtures/output/login/help-onprem.golden

```diff
@@ -40,7 +40,7 @@ Log in to Confluent Platform with a MDS URL and Certification Authority certific
 
   $ confluent login --url https://localhost:8090 --certificate-authority-path certs/my-cert.crt
 
-Log in to Confluent Platform with SSO and ignore any saved credentials.
+Log in to Confluent Platform with SSO and ignore login environment variables.
```

Comment:
We aren't ignoring any environment variables as far as I can tell. In fact, we require `CONFLUENT_PLATFORM_SSO` for this to work which we're definitely not ignoring.
```suggestion
Log in to Confluent Platform with SSO.
```

Reply:
The reason I struggled with this one is that the `CONFLUENT_PLATFORM_SSO` env var is used to bypass the two credential env vars. And users without any env vars set don't need it either to log in with SSO.

So how about something like
```
Log in to Confluent Platform with SSO even if `CONFLUENT_PLATFORM_USERNAME` and `CONFLUENT_PLATFORM_PASSWORD` are set.
```